exercises/change/example.go

```diff
@@ -0,0 +1,119 @@
+// Package change demonstrates making change with fewest number of coins.
+package change
+
+// Change returns a list of coins in increasing value order to make
+// change for target amount from the coins given; an infinite supply of coins is assumed.
+func Change(coins []int, target int) (result []int) {
+	solutions := make([][]int, 0)
+	if target == 0 {
+		result = []int{}
+	} else {
+		if target > 0 {
+			findSolutions(coins, target, &solutions)
+		}
+		if len(solutions) == 0 {
+			result = []int{-1}
+		} else if len(solutions) == 1 {
+			result = solutions[0]
+		} else {
+			result = chooseBestSolution(solutions)
+		}
+	}
```

Comment:
Yes, that is better. I will update.  If I add an error return it will look just a bit different.